plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplitManager.java

```diff
@@ -310,19 +311,22 @@ private static String buildSplitPath(String tableLocation, AddFileEntry addActio
     {
         // paths are relative to the table location and are RFC 2396 URIs
```

Comment:
pls adjust the comment - this is not anymore universally valid in case of shallow cloned tables.

Reply:
updated the comment